src/Generators/DocBlockGenerator.php

```diff
@@ -97,15 +108,14 @@ public function getGeneratedDocBlock()
      */
     protected function mergeGeneratedTagsIntoDocBlock($existingDocBlock)
     {
-        $docBlock = new DocBlock($this->removeExistingSupportedTags($existingDocBlock));
+        $docBlock = $this->docBlockFactory->create(($existingDocBlock ?: "/**\n*/"));
 
-        if (!$docBlock->getText()) {
-            $docBlock->setText('Class \\' . $this->className);
+        $summary = $docBlock->getSummary();
+        if (!$summary) {
+            $summary = sprintf('Class \\%s', $this->className);
```

Comment:
It looks like it might be here, something goes wrong and all classes are prefixed with a `\`